jboss-kie-kieserver/added/launch/kieserver-pull.sh

```diff
@@ -55,6 +55,8 @@ if [ "${KIE_SERVER_CONTAINER_DEPLOYMENT}" != "" ]; then
             exit $ERR
         fi
     done
+else
+    log_warning "Pull dependencies is disabled, skipping. Please make sure to provide all dependencies needed by the specified kjar."
```

Comment:
> >> info log about skipping because empty KIE_SERVER_CONTAINER_DEPLOYMENT

it is printed already:

```
WARN Warning: EnvVar KIE_SERVER_CONTAINER_DEPLOYMENT is missing.
WARN Example: export KIE_SERVER_CONTAINER_DEPLOYMENT='containerId(containerAlias)=groupId:artifactId:version|c2(n2)=g2:a2:v2'
```

Reply:
as it is already printed and covered by tests, marking this conversation as resolved